src/ConcreteOperations/convex_hull.jl

```diff
@@ -469,5 +469,10 @@ function monotone_chain!(points::Vector{VN}; sort::Bool=true
     # remove the last point of each segment because they are repeated
     copyto!(points, @view(lower[1:end-1]))
     copyto!(points, length(lower), @view(upper[1:end-1]))
-    return resize!(points, length(lower) + length(upper) - 2)
+    m = length(lower) + length(upper) - 2
+    if m == 2 && points[1] == points[2]
```

Comment:
For robustness (and consistency across the library), let's change this floating-point comparison `==`  to the corresponding one in `comparisons.jl`.

```julia
julia> p = [0.0, 0.0];

julia> q = [0.0 + eps(Float64), 0.0]
2-element Array{Float64,1}:
 2.220446049250313e-16
 0.0                  

julia> convex_hull([p, q, p, q])
1-element Array{Array{Float64,1},1}:
 [0.0, 0.0]

julia> convex_hull([p, q, p, q, p, q]) # this branch
2-element Array{Array{Float64,1},1}:
 [0.0, 0.0]                  
 [2.220446049250313e-16, 0.0]
```